### PR TITLE
Include wrap to camel-leveldb

### PR DESCRIPTION
**Pull request by blocha**
This pull request is continuation of following one: https://github.com/apache/camel/pull/2062 but for branch master.

To make pax exam integration tests works for everyone, wrap prerequisite should be added to camel-leveldb and/or other packages.

@davsclaus idea is to add wrap to camel-core which probably would solve problems with pax exam.

@oscerd @davsclaus What do you think?

### CODE REVIEW

**Comment by blocha** [comment]
@gnodet Not sure if this is what you wanted https://github.com/apache/camel/pull/2065/commits/27e318c6027748e3483a5331abd4805888afa42a

**Comment by oscerd** [comment]
@gnodet can you take a look to this updated PR? Thanks.

File: platforms/karaf/features/src/main/resources/features.xml [hunk 1]
```diff
@@ -1397,6 +1397,7 @@
     <bundle>mvn:org.apache.camel/camel-linkedin/${project.version}</bundle>
   </feature>
   <feature name='camel-leveldb' version='${project.version}' resolver='(obr)' start-level='50'>
+    <feature prerequisite="true">wrap</feature>
```
Comment (davsclaus) [inline platforms/karaf/features/src/main/resources/features.xml]: @gnodet I have not seen this attribute before ` prerequisite` in use in karaf features. Is this a new thing or in other words does it work on karaf 4.0.x onwards or 4.1.x onwards. And what does it really do?

**Comment by davsclaus** [comment]
We cannot do this as when you install in Karaf 4.1.3, you get this error

Caused by: org.xml.sax.SAXParseException: cvc-complex-type.3.2.2: Attribute 'prerequisite' is not allowed to appear in element 'feature'.

**Pull request closed without merging**